### DISCUSSION TITLE
Как удалить запись Контрагента?

### DISCUSSION THREAD

**Discussion by alphagamma5** [YetiForceCompany/YetiForceCRM#15429] [How to]
Поучилось так что
 я добавила одного контрагента, потом добавила поле ввода в карточку контрагента, а потом еще раз внесла изменения в карточку контрагента,
и теперь эта запись контрагента со стороны пользователя YetiForceCRM не открывается и не редактируется и не удаляется.

Как удалить запись Контрагента?

![image](https://user-images.githubusercontent.com/93891630/141121637-5ef2877a-8206-4b77-9261-68816dc5ea6b.png)

**Comment by vovpff** [comment]
Ничего неипонял, но очень интересно. Напишите на почту?

**Comment by alphagamma5** [comment]
короче, кажется я сломала базу контрагентов,

теперь чисто переустанавливать всё надо или можно как-то восстановить?

**Reply by mariuszkrzaczkowski** [reply]
it's best to create an issue with all the required information, because that's how no one will reply, because we can't tell fortunes from photos

**Comment by mariuszkrzaczkowski** [comment]
attach logs, the answer is in logs
